.gitignore

```diff
@@ -2,17 +2,5 @@ Cargo.lock
 compatibility_test/target/*
 */Cargo.lock
 target/*
-*__pycache__*
-*.mypy_cache*
-*.egg-info*
```

Comment:
Those should be left in in case some python module is tested locally and creates them